server/controllers/user/userPofile.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable camelcase */
+import { Request, Response } from 'express';
+import { ObjectId } from 'mongodb';
+import User from '../../database/Models/User';
+import { paramSchema } from '../../utils';
+
+const getUserById = async (req: Request, res: Response) => {
```

Comment:
There is a typo in the file name so fix it, and if you name it something else, since the interviews, reviews and info are all user profile.